eng/depProj.targets

```diff
@@ -47,6 +61,18 @@ See the LICENSE file in the project root for more information.
     <CopyNuGetImplementations>true</CopyNuGetImplementations>
   </PropertyGroup>
 
+  <Target Name="RemoveImplicitPackageReferences">
+    <ItemGroup>
+      <!-- Remove all implicit framework packages, depprojs are meant to be wholly explicit -->
+      <PackageReference Remove="@(PackageReference)" Condition="'%(PackageReference.IsImplicitlyDefined)' == 'true'" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="RunRestoreDuringBuild"
+          DependsOnTargets="Restore"
```

Comment:
I assume this is working around another issue?

Reply:
Actually it looks like this is how you made restore happen during normal build. I assume build uses ResolvePackageAssets target and that is where you are hooking the restore into?

Reply:
Yep, exactly.